mcr.cpp

```diff
@@ -29,20 +29,17 @@ int main(){
 				cout << "Player 1: ";
 			else
 				cout << "Player 2: ";
-			cout << "Which cell to mark? i:[1..3], j:[1..3]: ";
+			cout << "Which cell to mark? i:[0..2], j:[0..2]: ";
 			cin >> i >> j;
-			if (turn == false)
-			   game[i][j] = 'X';
-			else
-			   game[i][j] = 'O';
-			if (isWin(game)){
-				cout << "Win!" << endl;
-				break; // need to terminate the problem
+			while (i < 0 || i >2 || j < 0 || j > 2 || game[i][j] != ' '){
+                cout << "error! Please reenter the location!\n";
+                if (turn == false)
+                    cout << "Player 1: ";
+                else
+                    cout << "Player 2: ";
+                cout << "Which cell to mark? i:[0..2], j:[0..2]: ";
+                cin >> i >> j;
 			}
-		}
-	if (i==3) // all celles with i=0 to 2 have been inputted above but no winner yet
-	   cout << "Tie!" << endl;
 
-	// show the game to console
-	cout << game[0][0] << " " << game[0][1] << " " << game[0][2] << endl;
-	cout << game[1][0] << " " << 
+			if (turn == false)
+			   game[i][j] = 'X';
```

Comment:
it seems this patch delete some important code here

Reply:
you're right. I have fixed this issue